test/e2e/host_path.go

```diff
@@ -61,7 +61,9 @@ var _ = Describe("hostPath", func() {
 			namespace.Name)
 	})
 
-	It("should support r/w [Conformance]", func() {
+	// This test requires mounting a folder into a container with write privileges.
+	// Some clusters may not allow that.
+	It("should support r/w "+InsecureTestTag, func() {
```

Comment:
I'm wondering if this makes more sense as a feature tag, eg: `[Feature:WritableVolumes]`